packages/commonwealth/client/scripts/views/components/Avatar/AvatarUpload.tsx

```diff
@@ -103,6 +105,7 @@ export const AvatarUpload = ({
   useEffect(() => {
     // @ts-expect-error StrictNullChecks
     return () => files.forEach((file) => URL.revokeObjectURL(file.preview));
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
this is last resort, would love to figure out why this needs to be here

Reply:
seems like this `useEffect` doesn't do anything since the `files` are empty on mount. Will check if we can remove this effect entirly

Reply:
Yup, this was unnecessary, removed the useEffect here completely